ibis/bigquery/compiler.py

```diff
@@ -95,7 +95,10 @@ def _extract_field(sql_attr):
     def extract_field_formatter(translator, expr):
         op = expr.op()
         arg = translator.translate(op.args[0])
-        return 'EXTRACT({} from {})'.format(sql_attr, arg)
+        if sql_attr == 'epochseconds':
+            return 'UNIX_SECONDS({})'.format(arg)
+        else:
+            return 'EXTRACT({} from {})'.format(sql_attr, arg)
```

Comment:
You can use a f-string in the two statements. I know you copied from the previous code, but probably a good time to update it.